### ISSUE TITLE
Consumer status http response time degrades over time

### ISSUE THREAD

**Issue by endeepak** [linkedin/Burrow#164]
The status endpoint takes more than 3 minutes after leaving burrow running for ~10 hours. Same endpoint used to return within few seconds in the beginning. 

Additional Info: Our monitoring checks (sensu) make an http call to consumer status http endpoint every 5 minutes.

Curl output with timing information

```
# time curl -# localhost:8000/v2/kafka/kafka-local/consumer/test-consumer-group/status
{"error":false,"message":"consumer group status returned","status":{"cluster":"kafka-local","group":"test-consumer-group","status":"OK","complete":true,"partitions":[],"partition_count":1,"maxlag":null,"totallag":0},"request":{"url":"/v2/kafka/kafka-local/consumer/test-consumer-group/status","host":"burrow-node1","cluster":"kafka-local","group":"test-consumer-group","topic":""}}
real	3m41.631s
user	0m0.011s
sys	0m0.004s
```

There was no error logged in log file `/var/log/burrow/burrow.out` and the CPU usage was low during this call 

Burrow configuration

```
[kafka "kafka-local"]
....
zookeeper-path=/
zookeeper-offsets=true
offsets-topic=__consumer_offsets

[tickers]
broker-offsets=60

[lagcheck]
intervals=3
expire-group=604800
min-distance=1
zookeeper-interval=60
zk-group-refresh=300

[httpserver]
server=on
```

I don't have any experience in golang to understand fro codebase.  Is there any way to figure out why it takes > 3 minutes?

**Comment by jsmaupin** [comment]
I am seeing this as well.  We also query Burrow every 5 minutes, but many of our queries timeout even after 90 seconds. I am seeing low CPU and memory usage with both Burrow and ZooKeeper.  There is nothing printed to the logs.

**Comment by ackramer** [comment]
Same here, we have a metrics collector that pushes stats into OpenTSDB. Burrow will remain stable for long periods of time but will eventually have long timeouts for simple requests.

**Comment by toddpalino** [comment]
I haven't seen this for our installs, and we regularly run them for days and weeks without issue. However, we're also not querying the HTTP endpoints frequently (we're using the notifier to call out to another system). @endeepak, it may be useful to take a stack trace during one of these requests that is taking a long time. You can do this by sending a SIGABRT to Burrow, which will trigger it to crash and dump a panic to the out file.

**Comment by ackramer** [comment]
Not sure if this would help, but we hadn't configured logging and debug logs were enabled by default. Once we turned the log level down we stopped having performance issues and request times for the HTTP endpoint have been consistent since. We manage a cluster of 300k/req/s and 100 topics and the logs were going crazy. :)

Would be nice if Burrow had mentioned seelog / debug logs in more depth in the README, especially since a lot of folks don't know golang or seelog at all.

Hope this helps.
